setup/seleniumInstall.js

```diff
@@ -4,18 +4,21 @@ var path = require('path'),
   log = debug('nemo:log'),
   error = debug('nemo:error'),
   fs = require('fs'),
+  _ = require('lodash'),
   exec = require('child_process').exec;
 
 var seleniumInstall = function (version) {
   return function installSelenium(callback) {
-    //check package.json
-    var pkg = require(path.resolve(__dirname, '../package.json'));
-    if (pkg.dependencies['selenium-webdriver'] === version) {
+
+    var seleniumVersion = _.get(require('selenium-webdriver/package.json'), 'version');
```

Comment:
We do need to check selenium version used under `nemo` right? Is that is what is being used even if user  has a different version altogether?